### DISCUSSION TITLE
Welcome to calliope Discussions!

### DISCUSSION THREAD

**Discussion by brynpickering** [calliope-project/calliope#476] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sstroemer** [comment]
Hi, so maybe as an info why I am trying to learn more about the internal details: Stefan and I talked briefly about the possibility to include my research topics (applying decomposition to energy system optimization) in some kind of way with Calliope, so I quickly drafted a proof-of-concept, that exports to LP, reads that into a `JuMP` model and then applies a basic Benders decomposition to it.

That works, even if there are a lot of overheads when trying to do that blind, but I now need to understand the mathematical structure better to figure out how the decomposition could actually be helpful (e.g. the high amount of complicating variables intrigued me). Thanks for any insights (!), Stefan

**Comment by Nicolasost** [comment]
Hello everyone,
I'm working with Calliope for the first time. I have an attribute error, when I start the simulation. Has someone an idea how to solve it?
Kind regard, Nicolas

**Comment by Nicolasost** [comment]
<img width="463" alt="image" src="https://github.com/calliope-project/calliope/assets/167448330/fd82f761-de66-46d2-aa9f-91cee3edb4c0">
